source/new/condreorg.md

```diff
@@ -1,6 +1,6 @@
 # Cond-mat archive reorganization
 
-Because of the continued growth of the cond-mat e-print archive (see the [monthly totals](http://xxx.lanl.gov/year/cond-mat/96)), many people have desired a bit of reorganization. This is now implemented.
+Because of the continued growth of the cond-mat e-print archive (see the [monthly totals](http://xxx.lanl.gov/year/cond-mat/1996)), many people have desired a bit of reorganization. This is now implemented.
```

Comment:
Due to the continued growth of the cond-mat e-print archive (see the [monthly totals](http://xxx.lanl.gov/year/cond-mat/1996)), many people have desired a bit of reorganization. This is now implemented.